src/core/issues-notes.js

```diff
@@ -368,7 +368,14 @@ async function fetchAndStoreGithubIssues(github) {
 export async function run(conf) {
   const query = ".issue, .note, .warning, .ednote";
   /** @type {NodeListOf<HTMLElement>} */
-  const issuesAndNotes = document.querySelectorAll(query);
+  const allEls = document.querySelectorAll(query);
+
+  const issuesAndNotes = Array.from(allEls).filter(itm => {
+    // Ignores any elements that were selected that is an ancestor of the
+    // <svg> element.
+    return !itm.closest("svg");
```

Comment:
This check seems a bit expensive to me, as `closest()` will end up traversing until `<html>`. I think we can do the following safely:

```suggestion
    return !(itm instanceof HTMLElement);
```

![image](https://user-images.githubusercontent.com/8426945/184365068-40df0f56-98a2-4be7-a654-05f57f82c71b.png)

Reply:
One minor change -- it was `return itm instanceof HTMLElement;` instead of `return !(itm instanceof HTMLElement);`. Otherwise, *only* non-HTMLElement nodes were selected.